app/ui/legacy/src/main/java/com/fsck/k9/ui/K9Drawer.kt

```diff
@@ -179,6 +187,8 @@ class K9Drawer(private val parent: MessageList, savedInstanceState: Bundle?) : K
                 descriptionTextColor = selectedTextColor
                 selectedColorInt = drawerColors.selectedColor
                 icon = ImageHolder(createAccountImageUri(account))
+                badge = StringHolder(accountAndUnread.unreadCount.toString())
+                badgeStyle = BadgeStyle(R.drawable.empty, 0, 0, drawerColors.accentColor)
```

Comment:
We don't want to display `0` if there's no unread messages. And always using the accent color makes things a bit too colorful for my taste. I suggest changing the code to this:

```suggestion
                accountAndUnread.unreadCount.takeIf { it > 0 }?.let { unreadCount ->
                    badgeText = unreadCount.toString()
                    badgeStyle = BadgeStyle().apply {
                        textColorStateList = selectedTextColor
                    }
                }
```

Reply:
done